src/main/java/io/pinecone/clients/AsyncIndex.java

```diff
@@ -1039,6 +1068,124 @@ public ListenableFuture<ListResponse> list(String namespace, String prefix, Stri
         return asyncStub.list(listRequest);
     }
 
+    /**
```

Comment:
The reason why I added imports in AsyncIndex is because the operations are async and for data plane based.